lib/membrane/rtp/session_bin.ex

```diff
@@ -243,79 +254,68 @@ defmodule Membrane.RTP.SessionBin do
 
   @impl true
   def handle_pad_added(Pad.ref(:rtp_input, ref) = pad, ctx, %{secure?: true} = state) do
```

Comment:
can we merge the clauses for secure and not secure variants and conditionally link decryptors/encryptors?

Reply:
sure